code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -144,6 +147,10 @@
 		/*if(prob(25))
 			location.ReplaceWithSpace()
 			return 0*/
+
+	if(world.time - sound_start_time >= sound_length) //looping sound
+		playsound(src, "fire", 30, TRUE, 5, 3)
+		sound_start_time = world.time
```

Comment:
This will get expensive very fast. I'd prefer we didn't have this.

A cheaper alternative is to play the sound only once on areas that fire is in. You can use a unique addtimer for this.